pkgs/tools/audio/beets/builtin-plugins.nix

```diff
@@ -17,20 +17,32 @@
   };
   acousticbrainz.propagatedBuildInputs = [ python3Packages.requests ];
   albumtypes = { };
-  aura.propagatedBuildInputs = with python3Packages; [ flask pillow ];
-  badfiles.wrapperBins = [ mp3val flac ];
+  aura = {
+    propagatedBuildInputs = with python3Packages; [ flask pillow ];
+    testPaths = [ ];
```

Comment:
I'm not overly fond of having to list every single plugin which doesn't have tests but the only other option I could see was to modify `disabledTestPaths` in `preDist` via a shell script (removing all files which don't exist), which seemed a bit nasty.